backend/app/Http/Controllers/Web/Admin/DataImportController.php

```
@@ -0,0 +1,560 @@
+        // master-data file they cannot otherwise see.
+        return CsvImport::whereIn('entity', array_keys($this->registry->forSection($this->section())))
+            ->orderByDesc('id')
+            ->limit(self::HISTORY_ROWS)
+            ->get()
+            ->map(fn (CsvImport $i) => $this->importPayload($i, errorLimit: self::HISTORY_ERRORS));
+    }
+
+    /** @return array<int, string> */
+    private function userNames(): array
+    {
+        $ids = CsvImport::orderByDesc('id')->limit(self::HISTORY_ROWS)->pluck('user_id')->filter()->unique();
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\CsvImport::orderByDesc().


(staticMethod.notFound)